knative-operator/pkg/controller/knativeeventing/knativeeventing_controller.go

```diff
@@ -19,12 +19,17 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller"
 	"sigs.k8s.io/controller-runtime/pkg/handler"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
-// This needs to remain "knative-eventing-openshift" to be compatible with earlier versions.
-const finalizerName = "knative-eventing-openshift"
+const (
+	// This needs to remain "knative-eventing-openshift" to be compatible with earlier versions.
+	finalizerName = "knative-eventing-openshift"
+
+	requiredNsKey = "REQUIRED_EVENTING_NAMESPACE"
```

Comment:
Better use the suffix "EnvName", "requiredNsEnvName"  eg. to be aligned with the rest of the code base.